codegen/smithy-aws-typescript-codegen/src/main/java/software/amazon/smithy/aws/typescript/codegen/AddAwsAuthPlugin.java

```diff
@@ -72,39 +72,41 @@ public void addConfigInterfaceFields(
         SymbolProvider symbolProvider,
         TypeScriptWriter writer
     ) {
-        ServiceShape service = settings.getService(model);
-        if (!isSigV4Service(service) && isAwsService(service)) {
-            ServiceTrait serviceTrait = service.getTrait(ServiceTrait.class).get();
-            settings.setDefaultSigningName(
-                serviceTrait.getArnNamespace()
-            );
-            return;
-        }
+        if (!settings.getExperimentalIdentityAndAuth()) {
```

Comment:
Nit: Could invert this if statement to just short circuit if experimentalIdentityAndAuth is present